moveit_experimental/CMakeLists.txt

```diff
@@ -53,13 +69,19 @@ catkin_package(
     collision_detector_hybrid_plugin
     ${OCTOMAP_LIBRARIES}
   CATKIN_DEPENDS
+    message_runtime
     moveit_core
     geometric_shapes
     eigen_stl_containers
     eigen_conversions
     moveit_msgs
     kdl_parser
     pluginlib
+    # The following are needed by jog_arm:
+    moveit_ros_planning_interface
+    rosparam_shortcuts
```

Comment:
I don't think it belongs in ros-planning since its just a core ros utility that wraps rosparam server, but could be moved into some other ROS org like https://github.com/ros... at this point in the life of ROS1 I'm not convinced its worth the effort / OSRF will want the hassle.